scripts.js

```diff
@@ -0,0 +1,10 @@
+function showMenu() {
+  let div = document.querySelector('#links-container');
+  div.classList.toggle('active-menu');
+
+  let div2 = document.querySelector('#hamburger-btn');
+  div2.classList.toggle('switch-btn');
+
+  let div3 = document.querySelector('#header-logo');
+  div3.classList.toggle('active-logo');
+  }
```

Comment:
- [ ] According to the requirements when the user clicks (or taps) any of the mobile menu options (Links), the mobile menu disappears. so kindly implement it 👌

![image](https://user-images.githubusercontent.com/45231276/179664941-8af56f35-4565-4814-a33a-fe11bdb50449.png)